src/modules/launcher/Wox.Test/Plugins/WindowsIndexerTest.cs

```diff
@@ -214,5 +216,24 @@ public void WindowsSearchAPI_ShouldNotReturnResultsWithNullValue_WhenDbResultHas
             Assert.IsFalse(windowsSearchAPIResults.Any(x => x.Title == "file1.txt"));
             Assert.IsTrue(windowsSearchAPIResults.Any(x => x.Title == "file2.txt"));
         }
+
+        [Test]
+        public void ContextMenuLoader_ReturnContextMenuWithOpenInConsole_WhenLoadContextMenusIsCalled()
+        {
+            // Arrange 
+            var mock = new Mock<IPublicAPI>();
+            mock.Setup(api => api.GetTranslation(It.IsAny<string>())).Returns(It.IsAny<string>());
+            var pluginInitContext = new PluginInitContext() { API = mock.Object };
+            var contextMenuLoader = new ContextMenuLoader(pluginInitContext);
+            var searchResult = new SearchResult() { Path = "C:/", Title = "DummyFile.cs" };
+            var result = new Result() { ContextData = searchResult };
+
+            // Act
+            List<ContextMenuResult> contextMenuResults = contextMenuLoader.LoadContextMenus(result);
+
+            // Assert
+            Assert.AreEqual(contextMenuResults.Count, 2);
```

Comment:
[Suggestion] Instead of adding count, can we add the specific context menu items that should or shouldn't show. My reason behind saying this is that we are adding `run as admin` context menu item for apps returned by the indexer. If there are any such changes in the future regarding addition of buttons, the tests might fail. Also, we don't have tests for any of the other context menu items so it would be nice to test all of them, if they function as expected.

Reply:
That's a great suggestion! I will update the tests to directly check if a context menu is present.